sdk.yaml

```diff
@@ -286,6 +286,13 @@ Realtime:
       .synopsis: |
         Send a message on this channel.
         This is a message over the Realtime connection, not a REST operation.
+    State Events:
+      .specification:
+        [RTL2, TH1, TH2, TH3, TH4, TH5, RTN11d, RTL4e, RTL3a, RTL4g, RTL14, RTL24, RTP5]
+      .synopsis: |
+        `ChannelEvent` events can be received asynchronously when channel state changes.
+        The current `ChannelState` can be synchronously queried (e.g. via a `state` property on the channel instance).
```

Comment:
Is it necessary to classify the synchronous / asynchronous nature of these two things?

Reply:
I agree. Because this PR is at the bottom of a large stack I'm deferring the work to fix this under https://github.com/ably/features/issues/43.

I'm resolving this conversation on @lmars' behalf in order to allow me to merge this pull request quicker.